src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -264,70 +253,57 @@
             padding: 0;
             position: absolute;
             width: 1px;
-        }
 
-        input:focus + label {
-            outline: 2px solid black;
-        }
+            &:focus + label .fl-prefsEditor-switch {
+                outline: 2px solid #000000;
+                outline-offset: 1px;
+            }
 
-        input:checked + label .fl-prefsEditor-switch {
-            padding-left: 3em;
-            width: 2em;
-            background-color: #2da750;
-            box-shadow: -1em 1.1em 0.1em 0 rgba(172, 216, 92, 0.63) inset;
+            &:checked + label .fl-prefsEditor-switchOn,
+            &:not(:checked) + label .fl-prefsEditor-switchOff {
+                border-color: #000000;
+                font-weight: bold;
+                font-size: 1.2em;
 
-            .fl-prefsEditor-switch-inner {
-                top:-1.6em;
-                left: 0.46em;
+                .fl-prefsEditor-switch-indicator {
+                    visibility: visible;
+                }
             }
-        }
-        // With data attributes defining the on/off text, localizing the templates is easier
-        input + label {
-            [data-checkboxStateOn]:before {
-                content: attr(data-checkboxStateOn);
+
+            &:checked + label .fl-prefsEditor-switchOff
+            &:not(:checked) + label .fl-prefsEditor-switchOn {
+                border-color: transparent;
             }
-            [data-checkboxStateOff]:after {
-                content: attr(data-checkboxStateOff);
+        }
+
+        .fl-prefsEditor-panel-descr {
+            display: block;
+        }
+
+        .fl-prefsEditor-switch {
+            border: solid 2px #C7C7C8;
```

Comment:
@jhung I wasn't sure if we wanted the borders to increase in size as the text scaled. @dayotte did you have an intention for how they would behave with text size increases for example.